src/features/instantiate/containers/PartySelection.tsx

```diff
@@ -94,7 +94,7 @@ export const PartySelection = () => {
 
   const onSelectParty = async (party: IParty) => {
     await selectParty(party);
-    navigate('/'); // Back to Entrypoint.tsx, where the next step will be determined
+    navigate('/instance-selection');
```

Comment:
Instance selection is not always the path we want to go to after party selection. We should only show instance selection if that is configured (via `show === 'select-instance'`), so that's why we redirected back to the entrypoint earlier.